### DISCUSSION TITLE
How to perform standalone test suite

### DISCUSSION THREAD

**Discussion by polaris-alioth** [rabbitmq/rabbitmq-server#3856] [Questions] [answered]
When  i  performed test for rabbit module with 'make tests' command, it tooks 2h to complete these suites. It there any method  to perform standalone test suite.

**Comment by mkuratczyk** [comment, marked as answer]
https://github.com/rabbitmq/rabbitmq-server/blob/master/CONTRIBUTING.md#running-tests

**Reply by polaris-alioth** [reply]
Thanks very much!
